db/schema.rb

```diff
@@ -116,6 +116,24 @@
     t.index ["token"], name: "index_external_api_keys_on_token", unique: true
   end
 
+  create_table "integration_mapping_notifications",
+               id: :uuid,
+               default: -> { "gen_random_uuid()" },
+               force: :cascade do |t|
+    t.string "notifiable_type", null: false
+    t.uuid "notifiable_id", null: false
+    t.uuid "template_version_id", null: false
+    t.string "front_end_path"
+    t.datetime "processed_at"
+    t.datetime "created_at", null: false
+    t.datetime "updated_at", null: false
+    t.index %w[notifiable_type notifiable_id],
+            name: "index_integration_mapping_notifications_on_notifiable"
+    t.index ["template_version_id"],
+            name:
+              "index_integration_mapping_notifications_on_template_version_id"
+  end
+
```

Comment:
might need to rebase this on `develop` to get rid of these changes (as this PR went in already I think?)